composer.json

```diff
@@ -33,10 +33,10 @@
         }
     ],
     "require": {
-        "backpack/crud": "^4.1.0"
+        "backpack/crud": "^5.0"
```

Comment:
Just to make sure, in this release we are dropping support for v4 right?

Reply:
Yes!